services/scabbard/libscabbard/src/store/scabbard_store/diesel/operations/update_consensus_context.rs

```diff
@@ -63,16 +63,22 @@ impl<'a> UpdateContextAction for ScabbardStoreOperations<'a, SqliteConnection> {
                     let update_context =
                         Consensus2pcContextModel::try_from((&context, service_id))?;
 
-                    delete(
-                        consensus_2pc_context::table.filter(
-                            consensus_2pc_context::service_id.eq(format!("{}", service_id)),
-                        ),
-                    )
-                    .execute(self.conn)?;
-
-                    insert_into(consensus_2pc_context::table)
-                        .values(vec![update_context])
-                        .execute(self.conn)?;
+                    update(consensus_2pc_context::table)
+                        .filter(consensus_2pc_context::service_id.eq(format!("{}", service_id)))
```

Comment:
Note, instead of using `format` here, you can just call `service_id.to_string()`

Reply:
Updated, will make a PR in the future to update this elsewhere